src/test/java/com/wikia/webdriver/testcases/facebooktests/FacebookButtonTests.java

```diff
@@ -0,0 +1,79 @@
+package com.wikia.webdriver.testcases.facebooktests;
+
+import org.testng.annotations.Test;
+
+import com.wikia.webdriver.common.properties.Credentials;
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.pageobjectsfactory.componentobject.dropdowncomponentobject.DropDownComponentObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.WikiBasePageObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.signup.SignUpPageObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.special.SpecialNewFilesPageObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.special.login.SpecialUserLoginPageObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.special.preferences.PreferencesPageObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.special.preferences.PreferencesPageObject.tabNames;
+
+/* 
+ * Check for facebook button on the page
+ */
+
+public class FacebookButtonTests extends NewTestTemplate {
+
+	Credentials credentials = config.getCredentials();
+
+	// check for FB button on login drop down
```

Comment:
I could go either way in terms of these comments. I kinda like them, but the methods are well named so they may be redundant. If we do decide to keep them though, we should remove the space between the comment and method. eg 
```java
    // check for FB button on login drop down
 	@Test(groups = { "FBButton_001", "FacebookButton" })
	public void FBButton_001_DropDownButton_Visible() {
		DropDownComponentObject dropDown = new DropDownComponentObject(driver);
		dropDown.openDropDown();
		dropDown.verifyFBButtonVisible();
```
up to you @bekcunning.